packages/ts-endpoint-browser/src/__test__/fetch.spec.ts

```diff
@@ -296,4 +298,13 @@ describe('GetFetchHTTPClient', () => {
 
     expect((getResponse as any).left.details.meta).toEqual({ message: 'response is not a json.' });
   });
+  it('returns the modified error when using handleError', async () => {
+    global.fetch = jest.fn().mockReturnValueOnce(lazyServerErrorRequest());
```

Comment:
Tests are failing cause I think you need to _reject_ the promise for this implementation, in order to trigger the _left_ branch in TaskEither

Reply:
Thanks! I am on a linux laptop and I can't make lerna symlink my packages so I can't test locally...
Should probably try to understand why...

Reply:
Yep, you should probably fix that before :)